stdlib/encodings/__init__.pyi

```diff
@@ -1,7 +1,6 @@
-import codecs
-from typing import Any
+from codecs import CodecInfo
 
-def search_function(encoding: str) -> codecs.CodecInfo: ...
+class CodecRegistryError(LookupError, SystemError): ...
 
-# Explicitly mark this package as incomplete.
-def __getattr__(name: str) -> Any: ...
+def normalize_encoding(encoding: str) -> str: ...
+def search_function(encoding: str) -> CodecInfo: ...
```

Comment:
This can return None: https://github.com/python/cpython/blob/master/Lib/encodings/__init__.py#L119